File: basics/q1/solution.cpp
#include<iostream>
using namespace std;

void swap(int &a,int &b){
    int temp;
    temp = b;
    b = a;
    a = temp; 
    
}

int main(){
    int x = 5;
    int y = 7;
    cout<< x << y << endl;
    swap(x,y);
    cout<< x << y << endl;

    return 0;
}

File: basics/q1/solution.py
def swap(a,b):
     temp = a
     a = b
     b = temp

     return(a,b)

x = int(input("Enter first number: "))
y = int(input("Enter second number: "))
print("Before swap: ",x,y)
print("After swap: ",swap(x,y))